MAINTAINERS.md

```diff
@@ -12,11 +12,11 @@ This is the list of maintainers, including their email address for direct commun
 | Ryu Okada             | [@ryuo88](https://github.com/ryuo88)                 | okada@soramitsu.co.jp           | Product vision                                                                |
 | Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
 | Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
-| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Tech lead, development: Rust, C, C++                                          |
+| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Iroha 2 maintainer, previous tech lead, development: Rust, C, C++             |
```

Comment:
```suggestion
| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Development: Rust, C, C++             |
```
Just keep it simple

Reply:
Given the two uses, be it either finding people who contribute to the project or for Hyperledger administrative processes, I find my current idea to be valid.

Reply:
Commit history always can be an evidence for 'who contributes', please elaborate on the second use

Reply:
TOC elections are a good example here, even if Aleksandr is not a TOC member this time. I think leaving such information is helpful, but if you disagree, I'll remove it.

Reply:
> Just keep it simple

Agreed. Architect sounds more appropriate for what I did and continue to do.

Reply:
Besides, to be quite honest, I don't really need this document to showcase what I did. Anyone with half a brain will check the commit history, and it's more consistent with an architect than even a technical lead.